src/tests/test_json.c

```diff
@@ -87,6 +86,16 @@ MU_TEST(test_sprint)
   m = dlite_json_sprint(NULL, 0, inst, 4, dliteJsonSingle);
   mu_assert_int_eq(404, m);
 
+
+  /* Tests for proper quoting */
+  DLiteCollection *coll = dlite_collection_create(NULL);
+  dlite_collection_add_relation(coll, "s", "p", "\"o\"");
+  m = dlite_json_sprint(buf, sizeof(buf), (DLiteInstance *)coll, 2, 0);
+  printf("\n--------------------------------------------------------\n");
+  printf("%s\n", buf);
+  printf("\n--------------------------------------------------------\n");
+
+  //printf("\n========================================================\n");
```

Comment:
This is testing that you can print? But does it check what you print? Or is sprint testnig that you print the right thing?

Reply:
Merged in updated test from PR #633